ICRC-1.did

```diff
@@ -27,6 +27,7 @@ type TransferError = variant {
     TooOld : record { allowed_window_nanos : Duration };
     CreatedInFuture;
     Duplicate : record { duplicate_of : nat };
+    MaximumLoad;
```

Comment:
Maybe `TemporarilyUnavailable` or `ServiceUnavailable` to draw on existing HTTP expertise?

```suggestion
    TemporarilyUnavailable;
```

Reply:
sure